idearium-lib/test/query.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
```

Comment:
@allanchau, could you please make these tests a little more exhaustive? You should be testing `getOptions` and specifically the different incarnations of what you expect it to return.